origin-js/src/ipfsInterface/schemas/listing.json

```diff
@@ -54,11 +54,6 @@
     },
     "timeSlot": {
       "type": "object",
-      "required": [
```

Comment:
Good question... It's because the calendar has the concept of "unavailable" time slots, and a price isn't required in that case. Unavailable slots are used for overriding a repeating timeslot that is normally available, or for blocking out time on a calendar.
This could change in the future but for now, we can't make price required unless we force the dapp to put a price of `"0"` on all unavailable timeslots, which I'd be okay with, but this seemed cleaner IMO